### PR TITLE
Ecommerce product forms

### PR DESCRIPTION
**Pull request by chrisdanek**

### CODE REVIEW

File: marketplace_builder/views/pages/use-cases/e-commerce/creating-product-management-forms.liquid [hunk 1]
```diff
@@ -0,0 +1,1011 @@
...
+---
+name: admin_product_page
+redirect_to: /404
+---
+{% comment %} New product form {% endcomment %}
+{% if params.slug3 == 'new' %}
+  true
+{% comment %} Fetch product by id {% endcomment %}
+{% elsif params.slug3 != blank %}
+  {% query_graph "get_products", id: context.params.slug3, result_name: g %}
```
Comment (pavelloz) [inline marketplace_builder/views/pages/use-cases/e-commerce/creating-product-management-forms.liquid]:
```suggestion
  {% query_graph "get_products", id: context.params.slug3, result_name: "g" %}
```

File: marketplace_builder/views/pages/use-cases/e-commerce/creating-product-management-forms.liquid [hunk 2]
```diff
@@ -0,0 +1,1011 @@
...
+
+```liquid
+{% raw %}
+{% comment %}
+  Required params:
+    id: string
+{% endcomment %}
+
+{% comment %} Fetch product data {% endcomment %}
+{% query_graph "get_products", id: id, result_name: g %}
```
Comment (pavelloz) [inline marketplace_builder/views/pages/use-cases/e-commerce/creating-product-management-forms.liquid]:
```suggestion
{% query_graph "get_products", id: id, result_name: "g" %}
```

File: marketplace_builder/views/pages/use-cases/e-commerce/creating-product-management-forms.liquid [hunk 3]
```diff
@@ -0,0 +1,1011 @@
...
+{% endraw %}
+```
+
+Next, include field control in the form. You can use a shared partial, created in [Using Shared Partials](/use-cases/e-commerce/using-shared-partials) article.
+
+```liquid
+{% raw %}
+{%
+  include "forms/fields/text",
+    label: "Price",
```
Comment (pavelloz) [inline marketplace_builder/views/pages/use-cases/e-commerce/creating-product-management-forms.liquid]: Probably too deep indentation

File: marketplace_builder/views/pages/use-cases/e-commerce/creating-product-management-forms.liquid [hunk 4]
```diff
@@ -0,0 +1,1011 @@
...
+    }
+  {% endif %}
+---
+
+{% query_graph "get_brands", result_name: "gb" %}
+{% parse_json brand_options %}
+{
+  {% for brand in gb.customizations.results %}
+    {% unless forloop.first %},{% endunless %}
+    "{{ brand.id }}" : {{ brand.name | json }}
```
Comment (pavelloz) [inline marketplace_builder/views/pages/use-cases/e-commerce/creating-product-management-forms.liquid]:
```suggestion
    "{{ brand.id }}": {{ brand.name | json }}
```

File: marketplace_builder/views/pages/use-cases/e-commerce/creating-product-management-forms.liquid [hunk 5]
```diff
@@ -0,0 +1,1011 @@
...
+  {% endfor %}
+}
+{% endparse_json %}
+
+{% query_graph "get_product_types", result_name: "gpt" %}
+{% parse_json product_type_options %}
+{
+  {% for product_type in gpt.customizations.results %}
+    {% unless forloop.first %},{% endunless %}
+    "{{ product_type.id }}" : {{ product_type.name | json }}
```
Comment (pavelloz) [inline marketplace_builder/views/pages/use-cases/e-commerce/creating-product-management-forms.liquid]:
```suggestion
    "{{ product_type.id }}": {{ product_type.name | json }}
```

**Comment by diana-lakatos** [comment]
Thank you! Reviewed and edited.

**Review by diana-lakatos — APPROVED** [review]
Thank you!

**Pull request merged**